frontend/src/metabase/public/components/widgets/SharingPane/SharingPaneButton/SharingPaneButton.tsx

```diff
@@ -0,0 +1,35 @@
+import type { ReactNode } from "react";
+import {
+  SharingPaneButtonContent,
+  SharingPaneButtonTitle,
+} from "metabase/public/components/widgets/SharingPane/SharingPaneButton/SharingPaneButton.styled";
+import { Box, Center, Stack, Text } from "metabase/ui";
+
+type SharingOptionProps = {
+  illustration: () => JSX.Element;
+  children: ReactNode;
+  header: string;
+  description: ReactNode | string;
+  disabled?: boolean;
+};
+
+export const SharingPaneButton = ({
+  illustration,
+  children,
+  header,
+  description,
+  disabled,
+}: SharingOptionProps) => {
+  return (
+    <SharingPaneButtonContent disabled={disabled} withBorder>
+      <Center h="22.5rem" p="8rem">
```

Comment:
These values are from the design, maybe we can make them more responsive since the modal overflows on smaller screens